### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mhyryak**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -4,7 +4,44 @@ describe('getCoinCombination', () => {
...
+    expect(getCoinCombination(99)).toEqual([4, 0, 2, 3]);
+    expect(getCoinCombination(100)).toEqual([0, 0, 0, 4]);
+    expect(getCoinCombination(123)).toEqual([3, 0, 2, 4]);
+  });
+
+  test('handles edge cases correctly', () => {
+    expect(getCoinCombination(4)).toEqual([4, 0, 0, 0]);
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+    expect(getCoinCombination(25)).toEqual([0, 0, 0, 1]);
```
Comment (KarolVlasova) [inline src/getCoinCombination.test.js:44]: 1 test === 1 expect

**Review by KarolVlasova — APPROVED** [review]
